app/src/main/java/org/xbmc/kore/ui/sections/addon/AddonsActivity.java

```diff
@@ -20,76 +20,48 @@
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentTransaction;
-import android.support.v4.widget.DrawerLayout;
-import android.support.v7.app.ActionBar;
-import android.support.v7.widget.Toolbar;
 import android.view.Menu;
 import android.view.MenuItem;
-import android.view.Window;
 
 import org.xbmc.kore.R;
 import org.xbmc.kore.ui.AbstractFragment;
-import org.xbmc.kore.ui.BaseActivity;
-import org.xbmc.kore.ui.generic.NavigationDrawerFragment;
+import org.xbmc.kore.ui.BaseDrawerActivity;
 import org.xbmc.kore.ui.sections.remote.RemoteActivity;
 import org.xbmc.kore.utils.LogUtils;
-import org.xbmc.kore.utils.SharedElementTransition;
 import org.xbmc.kore.utils.Utils;
 
 /**
  * Controls the presentation of Addons information (list, details)
+ * Controls the presentation of Addons information (list, details)
  * All the information is presented by specific fragments
  */
-public class AddonsActivity extends BaseActivity
+public class AddonsActivity extends BaseDrawerActivity
         implements AddonListFragment.OnAddonSelectedListener {
     private static final String TAG = LogUtils.makeLogTag(AddonsActivity.class);
 
     public static final String ADDONID = "addon_id";
     public static final String ADDONTITLE = "addon_title";
-    public static final String LISTFRAGMENT_TAG = "addonlist";
 
     private String selectedAddonId;
     private String selectedAddonTitle;
 
-    private NavigationDrawerFragment navigationDrawerFragment;
+    @Override
+    protected Fragment createFragment() {
+        return null;
+    }
 
-    private SharedElementTransition sharedElementTransition = new SharedElementTransition();
+    @Override
+    protected String getActionBarTitle() {
+        return selectedAddonTitle;
+    }
 
-    @TargetApi(21)
     @Override
     protected void onCreate(Bundle savedInstanceState) {
-        // Request transitions on lollipop
-        if (Utils.isLollipopOrLater()) {
-            getWindow().requestFeature(Window.FEATURE_CONTENT_TRANSITIONS);
-        }
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_generic_media);
-
-        // Set up the drawer.
-        navigationDrawerFragment = (NavigationDrawerFragment)getSupportFragmentManager()
-                .findFragmentById(R.id.navigation_drawer);
-        navigationDrawerFragment.setUp(R.id.navigation_drawer, (DrawerLayout) findViewById(R.id.drawer_layout));
-
-        Fragment fragment;
         if (savedInstanceState == null) {
```

Comment:
I don't understand. If the bundle is not null this activity is recreated because of a configuration change. In which the fragment does not need to be recreated, right?